cpp/include/cudf/interop.hpp

```diff
@@ -31,6 +31,7 @@
 
 #include <cudf/column/column.hpp>
 #include <cudf/detail/transform.hpp>
+#include <cudf/interop/detail/arrow.hpp>
```

Comment:
I think this include can be moved to `to_arrow_device.cu`
And you should only need to forward reference the `ArrowDeviceArray` in this file by adding the following line before the `namespace cudf {` line:
```
struct ArrowDeviceArray;
```

Reply:
sure, I can clean this up with the forward declaration